dashboard/README.md

```diff
@@ -40,29 +40,38 @@ dashboards with different configurations.
 [replicator]: cmd/postgres_replicator/README.md
 [replicatorconfig]: config/postgres_replicator/default/config.yaml
 
+## Deploy dashboard with PSM
+
+PSM dashboard is not included in the deployment by default, following command is
+needed to deploy PSM dashboard together with OSS dashboard.
+
+```shell
+export PSM=true
+```
+
 ## Manual build
 
 Several environment variables must be set before building and deploying. The
 table below shows the names and values of the variables in our main dashboard:
 
-| Variable                    | Value                                         |
-| --------------------------- | --------------------------------------------- |
-| `BQ_PROJECT_ID`             | `grpc-testing`                                |
-| `CLOUD_SQL_INSTANCE`        | `grpc-testing:us-central1:grafana-datasource` |
-| `GCP_DATA_TRANSFER_SERVICE` | `postgres-replicator`                         |
-| `GCP_GRAFANA_SERVICE`       | `grafana`                                     |
-| `GCP_PROJECT_ID`            | `grpc-testing`                                |
-| `GRAFANA_ADMIN_PASS`        | ...                                           |
-| `PG_DATABASE`               | `datasource`                                  |
-| `PG_PASS`                   | ...                                           |
-| `PG_USER`                   | `grafana-user`                                |
+| Variable                    | Value                                   |
+| --------------------------- | --------------------------------------- |
+| `BQ_PROJECT_ID`             | `grpc-testing`                          |
+| `CLOUD_SQL_INSTANCE`        | `grpc-testing:us-central1:grafana-data` |
+| `GCP_DATA_TRANSFER_SERVICE` | `postgres-replicator`                   |
+| `GCP_GRAFANA_SERVICE`       | `grafana`                               |
+| `GCP_PROJECT_ID`            | `grpc-testing`                          |
+| `GRAFANA_ADMIN_PASS`        | ...                                     |
+| `PG_DATABASE`               | `datasource`                            |
+| `PG_PASS`                   | ...                                     |
+| `PG_USER`                   | `grafana-user`                          |
```

Comment:
I would add a line `PSM` | (see below).

Then below I would write "The dashboard for [PSM benchmarks](../index.md#psm-benchmarks) is only deployed if the `PSM` variable is set to a non-empty value, for instance `PSM=true`".

I would also update cloud_build.yaml to include the PSM variable, and then update the cloud build project to include `_PSM` with an appropriate value, for instance `true`.

Reply:
I have updated the cloud build project to include _PSM as true.